docs/concepts/05_credentials/02_ctypes.md

```diff
@@ -19,10 +19,37 @@ This data format is used to define [CType models](https://github.com/KILTprotoco
 The following are all required properties of the schema, with no additional properties allowed:
```

Comment:
This was not true then, and is less true now; there are also the `type: 'object'` and `additionalProperties: false` properties